README.md

```diff
@@ -1,85 +1,356 @@
+See [Phylum Risk Domains documentation](https://docs.phylum.io/docs/phylum-package-score#risk-domains) for more detail.
+
+**NOTE**: It is not enough to have the total project threshold set. Individual risk domain threshold values must be set,
+either in the UI or with `phylum-ci` options, in order to enable analysis results for CI. Otherwise, the risk domain is
+considered disabled and the threshold value used will be zero (0).
+
+There will be no note if no dependencies were added or modified for a given MR.
+If one or more dependencies are still processing (no results available), then the note will make that clear and the CI
+job will only fail if dependencies that have _completed analysis results_ do not meet the specified project risk
+thresholds.
+
+## Prerequisites
+
+The GitHub Actions environment is primarily supported through the use of a Docker image.
+The pre-requisites for using this image are:
+
+* Ability to run a [Docker container action][container]
+  * GitHub-hosted runners must use an Ubuntu runner
+  * Self-hosted runners must use a Linux operating system and have Docker installed
+* Access to the [phylumio/phylum-ci Docker image](https://hub.docker.com/r/phylumio/phylum-ci/tags)
+* A [GitHub token][gh_token] with API access
+  * Can be the default `GITHUB_TOKEN` provided automatically at the start of each workflow run
+    * Needs at least write access for `pull-requests` scope - see [documentation][scopes]
+  * Can be a personal access token (PAT) - see [documentation][PAT]
+    * Needs the `repo` scope or minimally the `public_repo` scope if private repositories are not used
+* A [Phylum token](https://docs.phylum.io/docs/api-keys) with API access
+  * [Contact Phylum](https://phylum.io/contact-us/) or create an account and register to gain access
+    * See also [`phylum auth register`](https://docs.phylum.io/docs/phylum_auth_register) command documentation
+  * Consider using a bot or group account for this token
+* Access to the Phylum API endpoints
+  * That usually means a connection to the internet, optionally via a proxy
+  * Support for on-premises installs are not available at this time
+* A `.phylum_project` file exists at the root of the repository
+  * See [`phylum project`](https://docs.phylum.io/docs/phylum_project) and
+    [`phylum project create`](https://docs.phylum.io/docs/phylum_project_create) command documentation
+
+[container]: https://docs.github.com/en/actions/creating-actions/creating-a-docker-container-action
+[gh_token]: https://docs.github.com/en/actions/security-guides/automatic-token-authentication
+[scopes]: https://docs.github.com/en/developers/apps/building-oauth-apps/scopes-for-oauth-apps#available-scopes
+[PAT]: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token
+
+## Supported lockfiles
+
+These are the currently supported lockfiles.
+They are grouped by language and also show the name of the tool that generates the lockfile.
```

Comment:
Are we going to remember to update this list? Seems like a link to `docs.phylum.io` would be more appropriate.

Reply:
Good point...will change.